languagetool-language-modules/ga/src/main/resources/org/languagetool/resource/ga/earraidi-ga.bs

```diff
@@ -0,0 +1,4697 @@
+Abbey An Mhainistir
```

Comment:
One-line comment and license would be great.

Reply:
I've deleted that (which I should've done about a year ago!)